common/src/main/java/com/wynntils/features/ExperienceGainMessageFeature.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright © Wynntils 2022-2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features;
```

Comment:
If the config category is Utilities, you should move the class to the corresponding package.